notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Configuration.scala

```diff
@@ -46,79 +41,81 @@ case class TopicCountsConfiguration (
 
 object Configuration {
 
-  private def fetchConfiguration(): Config = {
+  private def fetchConfiguration(workerName: String): Config = {
     val identity = AppIdentity.whoAmI(defaultAppName = "notification-worker")
     ConfigurationLoader.load(identity) {
-      case AwsIdentity(_, _, stage, _) => SSMConfigurationLocation(s"/notifications/$stage/workers")
+      case AwsIdentity(_, _, stage, _) => SSMConfigurationLocation(s"/notifications/$stage/workers/$workerName")
     }
   }
 
   def platform: Option[Platform] = Option(System.getenv("Platform")).flatMap(Platform.fromString)
 
+  def confPrefixFromPlatform: String = platform match {
+    case Some(p @ (Ios | Android | IosEdition | AndroidEdition)) => p.toString
+    case _ => throw new IllegalStateException("No Platform environment variable defined")
+  }
+
   private def jdbcConfig(config: Config) = JdbcConfig(
     driverClassName = "org.postgresql.Driver",
     url = config.getString("registration.db.url"),
     user = config.getString("registration.db.user"),
     password = config.getString("registration.db.password"),
-    maxConnectionPoolSize = config.getInt("registration.db.maxConnectionPoolSize")
+    maxConnectionPoolSize = 1
```

Comment:
This was of no value to put in the conf. The day we want more connection in a lambda we can edit the code. (but why would we want more than 1?)

Reply:
Yeah makes sense